src/python/pants/backend/jvm/tasks/coverage/cobertura.py

```diff
@@ -28,8 +28,6 @@ class Cobertura(BaseCoverage):
 
   @classmethod
   def register_options(cls, register, register_jvm_tool):
-    super(Cobertura, cls).register_options(register, register_jvm_tool)
```

Comment:
Since this extends something, it should continue to call `super`... if the logic in the base class is supposed to be completely disabled, then it is might not be a good target for an override, but rather an optional explicit call (see "liskov substitution principle") : ie "register_coverage_options()".